lib/Serialization/Serialization.cpp

```diff
@@ -1021,7 +1021,11 @@ void Serializer::writeInputBlock(const SerializationOptions &options) {
   allImportFilter |= ModuleDecl::ImportFilterKind::ImplementationOnly;
   allImportFilter |= ModuleDecl::ImportFilterKind::SPIAccessControl;
   SmallVector<ModuleDecl::ImportedModule, 8> allImports;
-  M->getImportedModules(allImports, allImportFilter);
```

Comment:
Is `allImportFilter` used for anything?

Reply:
Oops -- I forgot to delete it. Thanks for catching!